config/300-clusterimagepolicy.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright 2022 The Sigstore Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: clusterimagepolicies.cosigned.sigstore.dev
+spec:
+  group: cosigned.sigstore.dev
+  names:
+    kind: ClusterImagePolicy
+    plural: clusterimagepolicies
+    singular: clusterimagepolicy
+    categories:
+    - all
+    - sigstore
+    shortNames:
+    - cip
+  scope: Cluster
+  versions:
+  - name: v1alpha1
+    served: true
+    storage: true
+    schema:
+      openAPIV3Schema:
+        # This is the output of:
+        #   go run ./cmd/schema/ dump ClusterImagePolicy
```

Comment:
As #1417 progresses, we can simply rerun this command and paste its new output here.